aws-cli/loft-list

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -x
+
+LIST_FILE=aws-loft-list.json
+TAG_KEY=lab_type
+TAG_VALUE=loft-lab
+
+aws ec2 describe-instances --query 'Reservations[].Instances[].{PublicHostname:PublicDnsName,PublicIP:PublicIpAddress}' --filters "Name=instance-state-name,Values=running" "Name=tag:${TAG_KEY},Values=${TAG_VALUE}" --output json | jq --arg START 1 '($START | tonumber) as $s
+  | to_entries
+  | map({StudentID: ($s + .key), PublicHostname:.value.PublicHostname, PublicIP:.value.PublicIP })' > ${LIST_FILE}
```

Comment:
I tried out "Student ID" and "Public Hostname", and "Public IP" (note the spaces), and it worked for the text based json output. Not sure though, how that would translate into html though.